src/phases/scale/LLTPhases.yml

```diff
@@ -22,12 +23,75 @@ GlobalDefaults:
     - keys: {caid: 1, prid: 1, cuid: 1}
     - keys: {prid: 1, ts: 1, cuid: 1}
 
+  FiftyDocumentsList: &50DocumentList
```

Comment:
Is it the best we can do here? Appears that we are missing some kind of structure here- not sure what it could be here... [I guess it is rather an enhancement suggestion for genny then this code review]

Reply:
I couldn't see a better way than this (and this is how one of the other workloads does it).

It does look like a genny feature request.